frontend/src/locales/fr-CH-scout.json

```diff
@@ -4,10 +4,19 @@
       "name": "Fil rouge"
     }
   },
+  "components": {
+    "print": {
+      "printConfigurator": {
+        "config": {
+          "Story": "Fil rouge"
+        }
+      }
+    }
+  },
   "entity": {
     "user": {
       "fields": {
-        "nickname": "Nom de Scout"
+        "nickname": "Totem"
```

Comment:
Ah, you call the "scout name" totem?
cool

Reply:
Yes, but I don't know where it come from :) I thought in the German-speaking part you used the word "vulgo"

Reply:
Vulgo is a very old term. Today I only see it used in abbreviated form when stating your full name: Carlo Beltrame v/o Cosinus
In normal everyday language, we exclusively say Pfadiname in Swiss German :)

Reply:
Okay ! yes I only saw the v/o so by deduction I thought you were using the whole word